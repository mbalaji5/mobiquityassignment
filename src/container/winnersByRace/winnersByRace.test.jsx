import React from "react";
import { render } from "@testing-library/react";
import WinnersByRace from "./winnersByRace";

describe("Should valildate Winners ByRace", () => {
  it("Should validate shopshot", () => {
    const { baseElement } = render(<WinnersByRace />);
    expect(baseElement).toMatchSnapshot();
  });
});
