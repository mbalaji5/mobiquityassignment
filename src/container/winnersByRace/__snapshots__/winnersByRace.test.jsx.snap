// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Winners ByRace shopshot Should validate shopshot 1`] = `
<body>
  <div>
    <div>
      <div>
        ...Loading
      </div>
    </div>
  </div>
</body>
`;

exports[`Should valildate Winners ByRace Should validate shopshot 1`] = `
<body>
  <div>
    <div>
      <div>
        ...Loading
      </div>
    </div>
  </div>
</body>
`;
